src/pages/messages/TasksPage.tsx

```diff
@@ -129,13 +129,12 @@ const TasksPage = () => {
   const ticketTypeQuery =
     selectedTicketTypes.length > 0 ? `(${selectedTicketTypes.map((type) => 'type:' + type).join(' OR ')})` : '';
 
-  const selectedTicketStatuses = Object.entries(ticketStatusFilter)
-    .filter(([_, selected]) => selected)
-    .map(([key]) => key);
+  const selectedStatusesArray =
+    searchParams.get(TicketSearchParam.Status)?.split(',') ?? (ticketStatusValues as TicketStatus[]);
```

Comment:
Nå blir bare den ene typen (som vi vet er riktig fra før) castet til riktig type igjen
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/689c6644-3b83-4bf2-a3a7-6af60a1a1c52)
```suggestion
  const selectedStatusesArray = (searchParams.get(TicketSearchParam.Status)?.split(',') ?? ticketStatusValues) as TicketStatus[];
```

Reply:
samme gjelder andre også